amphimixis/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Core module that containы director with main API and modules that he uses"""
+
+from amphimixis.general import general
+from amphimixis import analyzer
+from amphimixis import configurator
+from amphimixis import builder
+from amphimixis import profiler
+from amphimixis import shell
+
+__all__ = [
+    "general",
+    "analyzer",
+    "configurator",
+    "builder",
+    "profiler",
+    "shell",
+]
```

Comment:
```suggestion
"""Core module that containы director with main API and modules that he uses"""

from amphimixis.general import general
from amphimixis import analyzer
from amphimixis import configurator
from amphimixis import builder
from amphimixis.profiler import Profiler
from amphimixis.shell import Shell

__all__ = [
    "general",
    "analyzer",
    "configurator",
    "builder",
    "Profiler",
    "Shell",
]
```

The `Shell` class is the only public symbol in the `shell` module, so you do not need to import the whole  module.
It allows to write `from amphimixis import Shell` and use class Shell directly without `shell.Shell`

The same applies to the profiler and maybe other modules too. Authors should provide info on their modules